triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerLogContext.scala

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger
+
+import com.daml.lf.engine.trigger.Runner.Implicits._
+import com.daml.logging.LoggingContextOf
+import com.daml.logging.LoggingContextOf.label
+import com.daml.logging.entries.LoggingValue.OfString
+import com.daml.logging.entries.{LoggingEntries, LoggingValue}
+
+import java.util.UUID
+
+object ToLoggingContext {
```

Comment:
Implicit here is designed to allow use of `TriggerLogContext` in areas where a `LoggingContextOf[Trigger]` is required